x-pack/platform/packages/shared/security/api_key_management/src/components/api_key_flyout.tsx

```diff
@@ -326,7 +326,7 @@ export const ApiKeyFlyout: FunctionComponent<ApiKeyFlyoutProps> = ({
 
   return (
     <FormikProvider value={formik}>
-      <EuiFlyout onClose={onCancel} aria-labelledby={titleId} size="m" ownFocus>
+      <EuiFlyout onClose={onCancel} aria-labelledby={titleId} size="s" ownFocus>
```

Comment:
Are we able to add a conditional here for breakpoints? So smaller breakpoints, it shows as `size="m"`, but larger breakpoints it changes to `"s"`?